gravitee-am-service/src/main/java/io/gravitee/am/service/impl/ProtectedResourceServiceImpl.java

```diff
@@ -300,6 +328,17 @@ public Single<Page<ProtectedResourcePrimaryData>> findByDomainAndTypeAndIds(Stri
                 });
     }
 
+    @Override
+    public Flowable<ProtectedResource> findByDomain(String domain) {
+        LOGGER.debug("Find protected resources by domainId={}", domain);
+        return repository.findByDomain(domain)
+                .onErrorResumeNext(ex -> {
+                    LOGGER.error("An error occurs while trying to find protected resources by domain {}", domain, ex);
```

Comment:
suggestion: though I know this is copy/pasted
```suggestion
                    LOGGER.error("An error occurred while trying to find protected resources by domain {}", domain, ex);
```
(and below)